lib/applications/resolvers.ts

```diff
@@ -1857,3 +1859,82 @@ export const updateApplicationPhysicianAssessment: Resolver<
 
   return { ok: true, error: null };
 };
+
+/**
+ * Delete an application
+ * @returns status of operation (ok)
+ */
+export const deleteApplication: Resolver<MutationDeleteApplicationArgs, DeleteApplicationResult> =
+  async (_parent, args, { prisma, logger }) => {
+    const id = args.input.id;
+
+    const application = await prisma.application.findUnique({
+      where: {
+        id,
+      },
+    });
+
+    if (!application) {
+      return { ok: false, error: `Application with ID ${id} not found` };
+    }
```

Comment:
Nit: can use the `rejectOnNotFound` option to throw an error if applicant is not found, this way we can include this fetch inside the try/catch below. Just a style thing, feel free to ignore

```suggestion
    const application = await prisma.application.findUnique({
      where: {
        id,
      },
      rejectOnNotFound: true,
    });

    if (!application) {
      return { ok: false, error: `Application with ID ${id} not found` };
    }
```

Reply:
`rejectOnNotFound` appears to be [deprecated](https://www.prisma.io/docs/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-5/rejectonnotfound-changes) in future Prisma versions, so I might keep this as is for maintainability.